cmake/pkgconfig.in

```diff
@@ -8,4 +8,4 @@ Version: v@OSRM_VERSION_MAJOR@.@OSRM_VERSION_MINOR@.@OSRM_VERSION_PATCH@
 Requires:
 Libs: -L${libdir} -losrm
 Libs.private: @ENGINE_LIBRARY_LISTING@
-Cflags: -I${includedir} -I${includedir}/osrm
+Cflags: -I${includedir} -I${includedir}/osrm -I@CMAKE_INCLUDE_PATH@ @CMAKE_CXX_FLAGS@
```

Comment:
Yes we should add a separate variable for that `DEPENDENCIES_INCLUDE_PATH` osr something. The problem is that CMAKE_INCLUDE_PATH will also inlcude paths relative to the `osrm-backend` source repository (e.g. `./include/osrm`, `./unit_tests`).